### ISSUE TITLE
Disable starting with system not working on Linux (8.x.x)

### ISSUE THREAD

**Issue by mehdi-abdelkhalek** [subhra74/xdm#1104]
Hi

Disabling "Launch XDM when system starts" does quite the opposite, XDM keeps starting with system anyway

1. Go to 'Settings'
2. Click on 'Advanced settings'
3. Uncheck 'Launch XDM when system starts'
4. Click on 'Save'

Expected behavior:
XDM shouldn't start with system

 - OS: Linux
 - XDM Version 8.x.x (8.0.29)

**Comment by vuxuanhungg** [comment]
Same here. Actually, the issue is that the state does not persist after you click "Save".

**Comment by misa1313** [comment]
I'm able to replicate the issue on version 8.0.29, using kernel 6.4.6 (Arch).

**Comment by Mycohl** [comment]
Temporary fix:

`echo > ~/.config/autostart/xdm-app.desktop`
`chmod a-w ~/.config/autostart/xdm-app.desktop`

This seems to be the only place it attempts to launch from on my Mint system.
After this, settings will still show "Launch XDM when system starts", but it won't actually load.
Note that this behavior has been present since the 7.x branch, so it may not ever be "fixed".

**Comment by colibris79** [comment]
> 
> `echo > ~/.config/autostart/xdm-app.desktop` `chmod a-w ~/.config/autostart/xdm-app.desktop`
> 
Thanks! That worked for me in Mint 21.3 . I was about to uninstall xdm. It's a shame it seems to be discontinued because it works great

**Comment by Mycohl** [comment]
> it works great

I respectfully disagree.

There are loads of sites where file download fails, or stalls indefinitely. (Huggingface?)

Loads of sites where video detection fails. (Any porn site?)

Loads of sites where the browser monitoring is bugged, so it tries to download every single image on the page, and pops up 15-20 download prompts. (Any modern blog?)

Yeah, it will never be fixed, and I honestly don't blame subhra74. What a pain it would be to fix all of that, just to have it break again several weeks later as baby-brained webdevs find new ways to bloat, maim, and destroy everything.

**Comment by colibris79** [comment]
> > it works great
> 
> I respectfully disagree.
> 

Yeah. I should have said "It used to work great" (before it was abandoned). So thanks subhra74, it was a nice tool for a while
